src/Components/Link/index.css

```diff
@@ -0,0 +1,45 @@
+
+    .Btn-Apply {
```

Comment:
Maybe you could keep the ident consistent here.

Reply:
This does not follow `SUIT CSS` naming conventions. Please take another look at the documentation [here](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md)

Reply:
It should be,  
```
.Btn--apply { }
```
Right?